jetson/src/core/robot.cpp

```diff
@@ -177,9 +179,9 @@ std::vector<Point3D> Robot::scan() {
 			moveServoArm(y, CARTESIAN_Y_MAX*j/3);
 			std::vector<Point3D> newFlowers = findFlowers();
 			flowersToVisit.insert(flowersToVisit.end(), newFlowers.begin(), newFlowers.end());
-			flowersToVisit = avgClusterCenters(flowersToVisit, 10);
 		}
 	}
+	flowersToVisit = avgClusterCenters(flowersToVisit, 10);
 	return flowersToVisit;
```

Comment:
i know i wrote this, but can also just return avgClusterCenters at this point don't need to reassign to flowersToVisit